package.json

```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "@hoodie/account": "^2.0.0",
     "@hoodie/admin": "^1.0.1",
-    "@hoodie/client": "^6.0.0",
+    "@hoodie/client": "^7.0.0",
```

Comment:
when we update `"dependencies"`, we use the `fix:` prefix to force a new release, this helps to deduplicate (sub) dependencies. Only for `"devDependencies"` we use the `chore:` prefix because they don’t get installed by modules that depend on it. 

But for this PR it makes no difference as we already have the `feat:` prefix, so I’m going ahead and merge it as is :)